src/features/governance/hooks/useVote.ts

```diff
@@ -1,5 +1,6 @@
 import { useCallback } from 'react';
 import { useAsyncCallback } from 'react-use-async-callback';
+import { MAX_AMOUNT_THRESHOLD } from 'src/config/consts';
```

Comment:
what's exactly the purpose of this const?

Reply:
to be fully transparent, I'm not entirely sure, but my understand is that it prevents using ALL your stcelo balance (I would run regularly into `NotEnoughStakedCelo` when using the full balance). It existed before for the `swapping` mechanism so I kept it for voting